superset-frontend/packages/superset-ui-chart-controls/src/shared-controls/components/ColumnConfigControl/constants.tsx

```diff
@@ -191,12 +204,14 @@ export const DEFAULT_CONFIG_FORM_LAYOUT: ColumnConfigFormLayout = {
       'columnWidth',
       { name: 'horizontalAlign', override: { defaultValue: 'left' } },
     ],
+    ['truncateLongCells'],
     ['d3TimeFormat'],
   ],
   [GenericDataType.BOOLEAN]: [
     [
       'columnWidth',
       { name: 'horizontalAlign', override: { defaultValue: 'left' } },
     ],
+    ['truncateLongCells'],
```

Comment:
I don't think we need this configuration for boolean, temporal and numeric columns:
1. They are usually short enough
2. They should display full values under any conditions anyway for data accuracy.

Reply:
So I originally was thinking that too and left it out for everything except string, but thought that it might be better to allow (since it is off by default anyway). Dates could be formatted in a way that's wider than what you want. I dunno, I'll leave it up to you. I haven't used it for anything other than string.

Reply:
@ktmud I'm going back through some of my old work and saw this. Do you definitely want the option stripped for everything other than strings?

Reply:
Yes, please disable this for other data types because the risk of data accuracy complaints outweighs the potential benefit. It just doesn't make sense to cut a number or state in the middle----it makes them incomprehensible.